ui/app/helpers/enforce-protocol.ts

```diff
@@ -0,0 +1,15 @@
+import { helper } from '@ember/component/helper';
+
+export function enforceProtocol(params/*, hash*/) {
+  let str = params[0];
+
+  let isHttps = str.indexOf('https://') !== -1;
+  let isHttp = str.indexOf('http://') !== -1;
```

Comment:
[sand] I’d suggest using [`String.prototype.startsWith`](http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) for these.

```suggestion
  let isHttps = str.startsWith('https://');
  let isHttp = str.startsWith('http://');
```